### ISSUE TITLE
Area query results different from Overpass Turbo

### ISSUE THREAD

**Issue by JamesChevalier** [wiktorn/Overpass-API#21]
Here's my query to collect all of the States within the US:

```
[out:json];
area['admin_level'='2']['name'='United States'];
(relation['admin_level'='4'](area););
out tags;
```

My Overpass server returns [Baja California](https://www.openstreetmap.org/relation/2589601) in the results for this query, even though this state is in Mexico. [Overpass Turbo](http://overpass-turbo.eu/s/LOo) correctly excludes this from the results.

I'm using the [wiktorn/overpass-api](https://hub.docker.com/r/wiktorn/overpass-api) Docker image that I started it with:

```
docker run \
  -e OVERPASS_META=yes \
  -e OVERPASS_MODE=clone \
  -e OVERPASS_DIFF_URL=https://planet.openstreetmap.org/replication/minute/ \
  -v /home/ubuntu/overpass_clone_db/:/db \
  -p 80:80 \
  -i -t \
  --name overpass_world \
  wiktorn/overpass-api
```

Is there something that I can do to correct these results?
Are there certain places I can look for signs of trouble?

I've also posted this in [StackOverflow](https://stackoverflow.com/questions/57646845/why-are-area-queries-in-my-own-overpass-server-returning-different-results-from), [GIS StackExchange](https://gis.stackexchange.com/questions/333815/why-are-area-queries-in-my-own-overpass-server-returning-different-results-from), and [OpenStreetMap Help](https://help.openstreetmap.org/questions/70615/why-are-area-queries-in-my-own-overpass-server-returning-different-results-from-the-official-server).

**Comment by JamesChevalier** [comment]
I've since went through the process of setting up Overpass API directly on an Ubuntu server (no Docker usage), and that server returned the correct/expected query results.

To be certain of whether there's any issue with this Docker image, I've deleted my old container & started the process back up again (with the same `docker run` command in my original post ☝️ ). That `docker run` step has completed, and I'm watching the `rules_loop.log` file after running `docker start overpass_world` for the area creation to finish.

With some luck, it'll Just Work and I can close this as a one-off issue that nobody needs to worry about. 🤞 👍
